stacker_blueprints/rds/base.py

```diff
@@ -399,14 +407,36 @@ def defined_variables(self):
         variables.update(additional)
         return variables
 
+    @property
+    def master_username(self):
+        variables = self.get_variables()
+        if self.is_snapshot_restore:
+            return NoValue
+        return variables.get("MasterUser")
```

Comment:
Also a little worried about allowing this to return a blank string - not sure if that will result in an error, but my guess is yes?

Reply:
It will -- added a check that it be not empty when not restoring from snapshot.